src/net/sf/memoranda/ui/EventsPanel.java

```diff
@@ -624,6 +659,81 @@ void monthRbEventB_actionPerformed(ActionEvent e) {
         exTable.initMonthTable(CalendarDate.today());
     }
     
+    /**
+     * Method: customRbEventB_actionPerformed()
+     * Inputs: ActionEvent e
+     * Returns: void
+     * 
+     * Description: When the custom day range radio button is selected, the custom range text
+     * field is displayed. US-101
+     */
+	void customRbEventB_actionPerformed(ActionEvent e) {
```

Comment:
Probably should be private unless there are plans to use this outside this class